dkist/io/asdf/resources/schemas/dataset-1.2.0.yaml

```
@@ -0,0 +1,50 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://dkist.nso.edu/schemas/dataset-1.2.0"
+
+title: |
+  A DKIST Level 1 Dataset object.
+description:
+  The container for a distributed dataset object.
+
+type: object
+properties:
+  data:
+    tag: "asdf://dkist.nso.edu/tags/file_manager-1.0.0"
+
+  wcs:
+    description: The coordinate system for the complete dataset.
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+
+  mask:
+    tag: "tag:stsci.edu:asdf/core/ndarray-1.*"
+
+  unit:
+    tag: "tag:stsci.edu:asdf/unit/unit-1.*"
+
+  meta:
+    description: Dataset metadata, describing the whole dataset.
+    type: object
+    properties:
+      headers:
+        description: A table of all the headers for the constituent files.
+        anyOf:
+          - tag: "tag:astropy.org:astropy/table/table-1.*"
+          - tag: 'tag:stsci.edu:asdf/core/ndarray-1.*'
+          - null: true
```

Given where we are ending up with this, this should only ever be a structured array.
```suggestion
          - tag: 'tag:stsci.edu:asdf/core/ndarray-1.*'
```